Gemfile.lock

```diff
@@ -2,79 +2,80 @@ GEM
   remote: https://rubygems.org/
   specs:
     SyslogLogger (2.0)
-    actionmailer (4.2.0)
-      actionpack (= 4.2.0)
-      actionview (= 4.2.0)
-      activejob (= 4.2.0)
+    actionmailer (4.2.5)
+      actionpack (= 4.2.5)
+      actionview (= 4.2.5)
+      activejob (= 4.2.5)
```

Comment:
I am not sure how this happened exactly, but I suspect this happened when I updated the sping gem, these could be dependencies for this version of spring, but I am not sure. Should I try to revert the update of the gems?